source/_integrations/number.markdown

```diff
@@ -33,7 +33,7 @@ The type of data a number represents impacts how it is displayed in the frontend
 - **data_rate**: Data rate in bit/s, kbit/s, Mbit/s, Gbit/s, B/s, kB/s, MB/s, GB/s, KiB/s, MiB/s, or GiB/s
 - **data_size**: Data size in bit, kbit, Mbit, Gbit, B, kB, MB, GB, TB, PB, EB, ZB, YB, KiB, MiB, GiB, TiB, PiB, EiB, ZiB, or YiB
 - **distance**: Generic distance in km, m, cm, mm, mi, yd, or in
-- **energy**: Energy in Wh, kWh or MWh
+- **energy**: Energy in Wh, kWh, MWh, MJ or GJ
 - **frequency**: Frequency in Hz, kHz, MHz or GHz
```

Comment:
```suggestion
- **energy**: Energy in Wh, kWh, MWh, MJ, or GJ
- **frequency**: Frequency in Hz, kHz, MHz, or GHz
```

Reply:
Oh sorry i went the wrong way then with the conflict, ill fix